block/manager.go

```diff
@@ -499,7 +500,15 @@ func (m *Manager) publishBlock(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		block.SignedHeader.Validators = types.ValidatorSet{Validators: []types.Validator{{PublicKey: pubKey}}}
+		// TODO: read staking query to construct validators
+		block.SignedHeader.Validators = &tmtypes.ValidatorSet{
```

Comment:
Is this just a placeholder for validators for now?

Reply:
Test